core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -808,6 +809,13 @@
 			),
 			'sql'                     => "int(10) unsigned NOT NULL default 0"
 		),
+		'formSettings' => array
+		(
+			'exclude'                 => true,
+			'inputType'               => 'checkbox',
+			'eval'                    => array('tl_class'=>'w50 clr', 'submitOnChange'=>true),
```

Comment:
Why do we need `clr` here? The widget is the first one.

Reply:
Fixed in 9e580618c